compiler/passes/createTaskFunctions.cpp

```diff
@@ -370,16 +370,21 @@ static void addReduceIntentSupport(FnSymbol* fn, CallExpr* call,
   Expr* tailAnchor = findTailInsertionPoint(call, isCoforall);
   // Doing insertAfter() calls in reverse order.
   // Can't insertBefore() on tailAnchor->next - that can be NULL.
-  tailAnchor->insertAfter("chpl__delete(%S)",
-                         globalOp);
-  tailAnchor->insertAfter("'='(%S, generate(%S,%S))",
```

Comment:
FYI, I think this is a case where the call-expr-string stuff actually made the bug harder to see. I definitely prefer the `new CallExpr` style, even though it can be more verbose.

Arguably the multi-arg version of insertAfter would be better to simply insert many things in order than to do this format string thing (My next comment has an example). I think the format string thing should have a different method name.